modules/mod_translation/mod_translation.erl

```diff
@@ -471,7 +460,7 @@ language_delete(LanguageCode, Context) ->
     end.
 
 %% @doc Remove a language from the i18n configuration
--spec remove_from_config(atom(), #context{}) -> undefined.
+-spec remove_from_config(atom(), #context{}) -> ok.
```

Comment:
The spec in line 463 uses a record, please define a type for the record and use that instead.